picky-server/src/db/mod.rs

```diff
@@ -64,6 +65,60 @@ pub struct CertificateEntry {
     pub key: Option<Vec<u8>>,
 }
 
+#[derive(Debug, Eq, PartialEq, Clone, Copy, Hash, Serialize, Deserialize)]
+pub enum SshKeyType {
+    Host,
+    Client,
+}
+
+impl SshKeyType {
+    pub fn as_str(&self) -> &str {
+        match &self {
+            SshKeyType::Client => "Client",
+            SshKeyType::Host => "Host",
+        }
+    }
+}
+
+impl ToString for SshKeyType {
+    fn to_string(&self) -> String {
+        match &self {
+            SshKeyType::Client => "Client".to_owned(),
+            SshKeyType::Host => "Host".to_owned(),
+        }
+    }
+}
+
+#[derive(Debug, Eq, PartialEq, Clone, Hash, Serialize, Deserialize)]
+pub struct SshKeyEntry {
+    pub key_type: SshKeyType,
+    pub key: String,
+}
+
+impl Default for SshKeyEntry {
+    fn default() -> Self {
+        SshKeyEntry {
+            key_type: SshKeyType::Host,
+            key: "".to_owned(),
+        }
+    }
+}
```

Comment:
it's just to satisfy `MemoryRepository<T>`'s `Debug` derive. It won't compile without it. 
An instance of `SshKeyEntry` is expected to be a valid key all time.

Reply:
You mean the `Default` derive? I see, you may just implement `Default` manually on the `MemoryRepository` instead (it contains a HashMap which could just be empty, we don't require an actual `SshKeyEntry` at any point, but the derive will force you to have a default implementation in case `T` is a normal field for instance).

Reply:
Yeah, the `Default derive. It's just already 11 PM in Ukraine :slightly_smiling_face:

Reply:
Haha! No trouble, don't forget to have some sleep :smile:

Reply:
May I resolve it?

Reply:
Of course, if you removed the `Default` impl from `SshKeyEntry`